libs/net/services/Helpers/ApiService.cs

```diff
@@ -333,19 +333,6 @@ public async Task<IEnumerable<TNO.API.Areas.Services.Models.Ingest.IngestModel>>
         var jsonString = JsonSerializer.Serialize(ingest);
         return await RetryRequestAsync(async () => await this.OpenClient.SendAsync<TNO.API.Areas.Services.Models.Ingest.IngestModel>(url, HttpMethod.Put, JsonContent.Create(ingest)));
     }
-
-    /// <summary>
-    /// Make an HTTP request to the api to update the ingest configuration.
-    /// </summary>
-    /// <param name="ingest"></param>
-    /// <returns></returns>
-    public async Task<TNO.API.Areas.Services.Models.Ingest.IngestModel?> UpdateIngestConfigAsync(
```

Comment:
How are you updating the ingest with the latest info if you've removed this?  I don't see the corresponding `UpdateIngestStateAsync` method

Reply:
`UpdateIngestStateAsync` already existed.  We already use it for updating `FailedAttempts`
I added a new method in the `IngestActionManager` specific to the new state value `UpdateIngestStateCreationDateOfLastItemAsync`